dev-docs/howtoguides/how_to_setup_windows_machine_for_wsl_testing.md

```diff
@@ -0,0 +1,181 @@
+# How to set up a Windows machine for WSL testing
+
+To run our integration tests on Windows Subsystem for Linux (WSL) instances, we need to first set up
+a Windows machine that will allow us to launch the WSL instances. After that, we also need to set up
+an OpenSSH server so we can run SSH commands on the Windows machine.
+
+This guide presents all the necessary steps for the set up, from launching the Windows 11 
+image in Azure to testing if WSL is correctly working on the machine.
+
+## Launching the image on Azure
+
+When launching the Windows instance on Azure, there are a few configurations that need to be performed,
+otherwise we will not be able to launch WSL instances.
+
+First of all, name the machine as `wsl-test`. Currently, this is the name we expect on the
+integration tests.
+
+After that, select the `Windows 11` image and mark the `Security Type` as *Standard*.
+![](windows-security-type.png)
```

Comment:
The screenshow shows Ubuntu not windows.
Also I see several `Windows 11`s
* Windows 11 Pro
* Windows 11 Pro N
* Windows 11 Enterprise
* Windows 11 Enterprise multi-session
* Windows 11 Enterprise N

I'll try out the Windows 11 Pro for now but it would be good to choose a specific one for the guide